src/material-examples/chips-stacked/chips-stacked-example.html

```diff
@@ -1,4 +1,4 @@
-<mat-chip-list class="mat-chip-list-stacked">
+<mat-chip-list class="mat-chip-list-stacked" aria-label="Example stacked chip">
```

Comment:
The labels should be related to what the user is selecting. E.g. this one could be something like "Pick some colors".

Reply:
As mat-chip-list role is listbox, "list" is getting appended to the label in screen readers audio. If I give aria label as "Pick some colors" it will become "Pick some colors list".

Reply:
On that case it should be something like "Color selection list" and "Fruit selection list".

Reply:
Changed as per your suggestion crisbeto